src/popover/popover.scss

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+
+// tippy.js helper class to hide Popover when it is scrolled out of view
+@mixin iui-popover {
+  &[data-reference-hidden] {
+    visibility: hidden;
```

Comment:
visibility is the expected solution? not `display: none` ?

Reply:
https://github.com/iTwin/iTwinUI/pull/464#discussion_r798674492

Reply:
I need to start reading previous comments :DD